errors.lua

```diff
@@ -367,7 +375,8 @@ local function netbox_call(conn, func_name, ...)
             ' (string expected, got ' .. type(func_name) .. ')', 2)
     end
 
-    return wrap_with_suffix(
+    return _wrap(
+        {'"%s:%d"', conn.host, conn.port},
```

Comment:
```
tarantool> c = netbox.connect(3301)
---
...

tarantool> c
---
- peer_uuid: 7adcb555-3460-46a3-a342-6c1af6385b2f
  schema_version: 271
  protocol: Binary
  state: active
  peer_version_id: 132355
  port: '3301'
...

tarantool> c.host
---
- null
...

tarantool> c.port
---
- '3301'
...
```

Such case will lead to `"nil:3301"` prefix

Reply:
Maybe it's ok? Because even `require('uri')` doesn't handle it:

```
tarantool> uri.format({service = '3301'})
---
- 
...

tarantool> uri.format({host = 'abcd', service = '3301'})
---
- abcd:3301
...
```

What would you expect to see in this case?

- `NetboxEvalError: ":3301": Artificial error`?
- `NetboxEvalError: "localhost:3301": Artificial error`?
- `NetboxEvalError: 3301: Artificial error`?

Should I remove quotes at all?

Reply:
The first is ok. Also maybe I was a bit inattentive, in my example port is a string. And "%d" in format will throw in such case. 


> Should I remove quotes at all?

IMO, no, let's stay as is.

Reply:
Fixed